### DISCUSSION TITLE
Elastalert for logs ingested to ElasticSearch cluster per day that is size?

### DISCUSSION THREAD

**Discussion by ishukeshri2712** [jertel/elastalert2#1496] [Q&A] [answered]
Is there any way we can send alerts through mail how much logs are ingested to Elastic search ?

**Comment by jertel** [comment, marked as answer]
Consider an `any` type rule with no filter that runs once a day and has `use_count_query: true`. You might then have access to `num_hits` as a var in your alert message. I haven't tried this, so you'll need to read the docs and play with it to see what happens.
